src/wallet/wallet2.h

```diff
@@ -1008,6 +1008,22 @@ namespace tools
     crypto::public_key get_multisig_signing_public_key(size_t idx) const;
     crypto::public_key get_multisig_signing_public_key(const crypto::secret_key &skey) const;
 
+    template<class t_request, class t_response>
```

Comment:
I believe each of these 3 methods should acquire m_daemon_rpc_mutex. No?

Reply:
Good catch!